tools/cli/services/create.js

```diff
@@ -1,25 +1,70 @@
-const fs = require('fs');
-const execute = require('../utils/execute');
-const { red } = require('../utils/colors');
-const { join } = require('path');
+const fs = require("fs");
+const execute = require("../utils/execute");
+const { red } = require("../utils/colors");
+const { join } = require("path");
+
+function runNpmInstallInDirectoryAndExitIfFailed(directoryName) {
```

Comment:
Please find a shorter name. I believe "installDependencies" is enough.

Reply:
About a week ago, I would probably name this function something similar to what you suggested. However, I have just finished reading "Clean Code" by Robert C. Martin, and in the book the author suggests that function names (and names in general) should be descriptive and tell exactly what a function does. I have come to agree with author, and although the function name is long, I think it is better this way, since it describes exactly what that function does.

I'd love to hear what you think about this, and whether you would rather me renaming the function anyway.

Reply:
it's too descriptive.
there's a difference between:
`function sum(a, b) {return a + b}`
to:
`addTheFirstNumberToTheSecondNumberAndReturnASumOfBoth(a, b) { return a + b}`

the first one is also descriptive, and even better than the second.